src/logic.py

```diff
@@ -139,63 +164,122 @@ def __straight_trajectory(self, tank: Tank) -> tuple[Optional[Hex], Optional[lis
         tanks_shot_at = []
 
         # Add to tanks_shot_at
-        coord = tank_shot.get_position()
-        d = Hex.distance(tank.get_position(), coord)
+        coord = tank_shot.position
+        d = Hex.distance(tank.position, coord)
         (dx, dy, dz) = ((x - coord.q) / d, (y - coord.r) / d, (z - coord.s) / d)
-        for i in range(tank.get_max_range()):
+        for i in range(tank.max_range):
             new_hex = Hex(x - (i + 1) * dx, y - (i + 1) * dy, z - (i + 1) * dz)
             if new_hex in shoot_coords:
                 for t in enemy_tanks:
-                    if new_hex == t.get_position():
+                    if new_hex == t.position:
                         tanks_shot_at.append(self.__tank_from_hex(new_hex))
 
         return Hex(x - dx, y - dy, z - dz), tanks_shot_at
 
     def __shoot_coords(self, tank: Tank) -> list[Hex]:
         # coords based on center of map
         general_shoot_coords = []
-        for radius in range(tank.get_min_range(), tank.get_max_range() + 1):
+        for radius in range(tank.min_range, tank.max_range + 1):
             general_shoot_coords += Hex.hex_ring(Hex(0, 0, 0), radius)
 
         # if at_spg keep only straight lines
-        if tank.get_type() == "at_spg":
+        if tank.type == "at_spg":
             general_shoot_coords = [coord for coord in general_shoot_coords if 0 in coord]
 
         # get positional coord based on current tank position and remove those that are not in map
-        x, y, z = tank.get_position()
+        x, y, z = tank.position
         positional_shoot_coords = [Hex(x + dx, y + dy, z + dz) for (dx, dy, dz) in general_shoot_coords
                                    if not self.__offTheGridDetection(Hex(x + dx, y + dy, z + dz))]
 
         return positional_shoot_coords
 
     def __neutrality_check(self, tank_1: Tank, tank_2: Tank) -> bool:
-        id1, id2 = tank_1.get_player_id(), tank_2.get_player_id()
+        id1, id2 = tank_1.player_id, tank_2.player_id
         id3 = self.__third_player(id1, id2)
-        if id1 in self.__map.get_shoot_actions()[id2] or id2 not in self.__map.get_shoot_actions()[id3]:
+        if id1 in self.__map.shoot_actions[id2] or id2 not in self.__map.shoot_actions[id3]:
             return False
         else:
             return True
 
     def __third_player(self, id1: int, id2: int) -> int:
-        for p in self.__map.get_players():
+        for p in self.__map.players:
             if p.id != id1 and p.id != id2:
                 return p.id
 
     def reset_shoot_actions(self, player_id: int) -> None:
-        self.__map.get_shoot_actions()[player_id] = []
+        self.__map.shoot_actions[player_id] = []
 
     def is_in_base(self, tank_pos: Hex) -> bool:
-        return tank_pos in self.__map.get_base()
+        return tank_pos in self.__map.base
 
     # Detects if the given coordinates are off the grid
     def __offTheGridDetection(self, h: Hex) -> bool:
```

Comment:
why is the codestyle different here?

Reply:
Will be fixed in no time. Initially, two of us where working on these methods, so I forgot to change it